### PR TITLE
Add phoenix wings and connect to the websocket

### PR DESCRIPTION
**Pull request by hoarf**
This MR adds phoenix_wings as dep and sets up a basic connection with the socket.

### CODE REVIEW

**Comment by execb5** [comment]
que código maravilhoso

File: lib/screens/home_screen.dart [hunk 1]
```diff
@@ -1,5 +1,10 @@
 import 'package:flutter/material.dart';
 import 'package:cdparty_flutter/screens/tag_screen.dart';
+import 'package:phoenix_wings/phoenix_wings.dart';
+
+final socket = new PhoenixSocket(
+    "ws://10.0.2.2:4000/socket/websocket?token=alan",
+    socketOptions: PhoenixSocketOptions(params: {"token": "alan"}));
```
Comment (amarantedaniel) [inline lib/screens/home_screen.dart:8]: do we need to specify the token twice?

**Comment by amarantedaniel** [comment]
> This MR adds phoenix_wings as dep and sets up a basic connection with the socket.

takes these broken wings and learn to fly

File: lib/screens/home_screen.dart (hunk 1 above)
Comment (hoarf) [reply lib/screens/home_screen.dart:8]: no, the first one can be removed

File: lib/screens/home_screen.dart [hunk 2]
```diff
@@ -1,5 +1,10 @@
 import 'package:flutter/material.dart';
 import 'package:cdparty_flutter/screens/tag_screen.dart';
+import 'package:phoenix_wings/phoenix_wings.dart';
+
+final socket = new PhoenixSocket(
+    "ws://10.0.2.2:4000/socket/websocket?token=alan",
```
Comment (amarantedaniel) [inline lib/screens/home_screen.dart:6]: please remove redundant token

**Pull request merged**